appendices/editor-in-chief-checks.md

```diff
@@ -9,12 +9,12 @@ to work on them before the review process begins.
 Please check our [Python packaging guide](https://www.pyopensci.org/python-package-guide) for more information on the elements
 below.
 
-- [ ] **Installation** The package can be installed from a community repository such as PyPI (preferred), and/or a community channel on conda (e.g., conda-forge, bioconda).
+- [ ] **Installation** The package can be installed from a community repository such as PyPI (preferred), and/or a community channel on conda (for example, conda-forge, bioconda).
```

Comment:
should an explicit mention be made of other forges like those on `pixi` (e.g.https://prefix.dev/channels/rg-forge) 

These are a little different, in that they don't really have any kind of community validation.

Reply:
@all-contributors please add @HaoZeke for docs, review